tests/OmniSharp.Roslyn.CSharp.Tests/FindReferencesFacts.cs

```diff
@@ -228,6 +230,62 @@ public class Foo {
             Assert.Equal("a.cs", usages.QuickFixes.ElementAt(0).FileName);
         }
 
+        [Fact]
+        public async Task CanFindReferencesFromMetadata()
```

Comment:
that's a good point - I have applied this change to the other test. This one I removed completely cause it was a left over from when I wanted to include other features beyond Go To Definition in this PR (they'll be handled by separate PRs).